dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/oracle/OracleMetadataConnector.java

```diff
@@ -198,22 +91,18 @@ public MessageTask(@Nonnull GroupTask... ts) {
     @Override
     protected Void doRun(TaskRunContext context, @Nonnull ByteSink sink, @Nonnull Handle handle)
         throws Exception {
-      int c = 1;
       LOG.error("All the select tasks failed:");
-      for (GroupTask task : tasks) {
+      for (int i = 1; i <= tasks.length; i++) {
```

Comment:
Is this loop correct? We index onto the tasks array, which should go from 0 to i < tasks.length, not 1?

Reply:
Thanks for spotting this bug, this is fixed now. I removed array index operations too.